### PR TITLE
Caches police id list

### PR DESCRIPTION
**Pull request by jbarnsley10**

### CODE REVIEW

File: app/repos/user-accounts.js [hunk 1]
```diff
@@ -345,10 +345,6 @@ const isAccountEnabled = async (username) => {
  * @param {string} username
  * @return {Promise<UserAccount>}
  */
-
-/**
- * @type {GetAccount}
- */
```
Comment (whitewaterdesign) [inline app/repos/user-accounts.js:351]: this is used in the export

File: app/routes/search-index.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports = [{
       if (!cachedPage?.success) {
         await auditSearch(request.params.terms, request.query, user)
 
-        const results = await search(user, request.params.type, request.params.terms, !!request.query.fuzzy, !!request.query.national)
+        const results = await search(request, user, request.params.type, request.params.terms, !!request.query.fuzzy, !!request.query.national)
```
Comment (whitewaterdesign) [inline app/routes/search-index.js:35]: not a big one and happy to be proved wrong, but if you are passing request, are the other arguments still needed?  They may be?

File: app/search/search-processors/search-police-list.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  return null
+}
+
+const saveListToCache = async (user, request, policeIds) => {
+  const cacheKey = buildCacheKey(user)
+  const now = new Date()
+  await set(request, cacheKey, { policeIds, expiry: addMinutes(now, expiryPeriodInMins) }, expiryPeriodInMins * MINUTE)
+  return policeIds
+}
+
```
Comment (whitewaterdesign) [inline app/search/search-processors/search-police-list.js:36]: wondering if it would make sense to move this to a cache folder?

**Review by whitewaterdesign — CHANGES_REQUESTED** [review]
Looks good, just a couple of questions / small changes.

File: app/search/search-processors/search-police-list.js (hunk 3 above)
Comment (jbarnsley10) [reply app/search/search-processors/search-police-list.js:36]: I have two mechanisms (search-results-paginator and search-police-list) which use caching functions. I'd prefer to keep them in the search folder (as they are closely tied to search functionality), and keep the cache folder to hold the base cache methods

**Review by whitewaterdesign — APPROVED** [review]
LGTM

**Pull request merged**